packages/synapse-interface/components/Portfolio/PortfolioTokenAsset.tsx

```diff
@@ -39,19 +45,62 @@ const handleFocusOnInput = () => {
   inputRef.current.focus()
 }
 
+function checkCCTPChainConditions(
+  fromChainId: number,
+  toChainId: number
+): boolean {
+  switch (true) {
+    case fromChainId === 1 && toChainId === 42161:
+      return true
+    case fromChainId === 42161 && toChainId === 1:
+      return true
+    case fromChainId === 1 && toChainId === 43114:
+      return true
+    case fromChainId === 43114 && toChainId === 1:
+      return true
+    case fromChainId === 42161 && toChainId === 43114:
+      return true
+    case fromChainId === 43114 && toChainId === 42161:
+      return true
+    default:
+      return false
+  }
+}
```

Comment:
also think the whole thing can be written with a Set

```
function checkCCTPChainConditions(fromChainId: number, toChainId: number): boolean {
  const CctpPairs = new Set([
    `${ETH.id}-${ARBITRUM.id}`, `${ARBITRUM.id}-${ETH.id}`, 
    `${ETH.id}-${AVALANCHE.id}`, `${AVALANCHE.id}-${ETH.id}`, 
    `${ARBITRUM.id}-${AVALANCHE.id}`, `${AVALANCHE.id}-${ARBITRUM.id}`
  ]);

  return CctpPairs.has(`${fromChainId}-${toChainId}`);
}
```

Reply:
or could be just a list with an includes

Reply:
great suggestion, this way is alot more elegant and scalable! updated in https://github.com/synapsecns/sanguine/pull/1150/commits/d1d60246b3fb20fea9b28620b0cff327cc0a8a7e